cluster/environments/azure-single-keyvault/variables.tf

```diff
@@ -38,6 +38,10 @@ variable "gitops_ssh_url" {
   type = "string"
 }
 
+variable "gitops_url_branch" {
+  type = "string"
+}
```

Comment:
I would suggest adding a default value to keep configurable values in `.tfvars` at minimum.